src/timing.jl

```diff
@@ -9,9 +9,8 @@ export every, fpswhen, fps, timestamp
 function every(delta::Float64)
     i = Input(time())
     update(timer) = push!(i, time())
-    t = Timer(update)
-    start_timer(t, delta, delta)
-    return lift(x->x, i, typ=Float64) # prevent push!
+    t = Timer(update, delta, delta)
+    return lift(identity,  i) # prevent push!
```

Comment:
> prevent push!
Do we need the same mechanism for fpswhen?
I'm not sure how this prevents "push!"